src/app/complex/complex-results/complex-results.component.ts

```diff
@@ -136,6 +137,21 @@ export class ComplexResultsComponent implements OnInit, AfterViewInit {
     this.reloadPage();
   }
 
+  public onComplexNavigatorGroupingChange(grouping: NavigatorComponentGrouping) {
```

Comment:
Delete these 3 methods. No longer needed after the other changes mentioned on other comments.